components/Icon.tsx

```diff
@@ -0,0 +1,50 @@
+type Props = {
+  type: string;
```

Comment:
you can make this more strict

```suggestion
  type: 'search' | 'home';
```